apps/project/tests/test_apis.py

```diff
@@ -798,6 +798,18 @@ def test_options(self):
         response = self.client.get(url)
         self.assert_200(response)
 
+    def test_project_status_in_project_options(self):
+        url = '/api/v1/project-options/'
+
+        self.authenticate()
+        response = self.client.get(url)
+        self.assert_200(response)
+        self.assertIn('project_status', response.data)
+        self.assertEqual(response.data['project_status'][0]['key'], 'active')
+        self.assertEqual(response.data['project_status'][0]['value'], 'Active')
+        self.assertEqual(response.data['project_status'][1]['key'], 'inactive')
+        self.assertEqual(response.data['project_status'][1]['value'], 'Inactive')
```

Comment:
Let's use variables instead of raw values.


```python3
from django.utils.hashable import make_hashable

choices = dict(make_hashable(Project.STATUS_CHOICES))

# For `active`/`inactive` let's use Project.ACTIVE Project.INACTIVE

# For `Active`/`Inactive` let's use choices[Project.ACTIVE] / choices[Project.INACTIVE]
```

Reply:
Done, Thanks